### ISSUE TITLE
Create easy way for team admins to navigate to team user management page

### ISSUE THREAD

**Issue by lowellrex** [department-of-veterans-affairs/caseflow#8170]
There is currently no way for admins of the various teams at the Board (and VSOs) to navigate to the user management page. This ticket exists to do the engineering work required to add that link/dropdown/whatever other method we decide on to navigate to the user management page.

_Current look at the user management page:_
![image](https://user-images.githubusercontent.com/32683958/49739815-c5c6cc00-fc60-11e8-8ef8-cde718c43e9e.png)

## Acceptance criteria
* [ ] Team admins and judges see an option in their top dropdown menu with the copy "[Team name] management"
* [ ] Team admins or judges click that link, they navigate to the user management page (`/organizations/vlj-support-staff/users`)

**Comment by laurjpeterson** [comment]
@lowellrex - I connected with @allyceh and @sneha-pai on this and updated the AC above. would love your input on if you think using the Switch Views dropdown will work for this for now.

**Comment by lowellrex** [comment]
Sure! I think using the "Switch Views" dropdown will definitely work for now

**Comment by joeyyang** [comment]
couple questions: 
- how do we tell if someone is a team admin?
- we are putting this dropdown both on the user management page (`/organizations/:organization/users`) and [somewhere else? anywhere else where the switcher lives?]

**Comment by laurjpeterson** [comment]
I can answer your second question: Yes, we should put it everywhere the switcher lives. I think this is individual queues, organization queues, and the user management page.

**Comment by lowellrex** [comment]
> how do we tell if someone is a team admin?

The `OrganizationsUser` table has an `admin` field that tells us if somebody is an admin of a team. We can use the `User.administered_teams` method when we build the list of `selectable_organizations` to determine the URL we should be pointing to (though we will probably have to move the serialized object [we build in the template file](https://github.com/department-of-veterans-affairs/caseflow/blob/0590e73effa48112a2a340d740ad6cb23901268e/app/views/queue/index.html.erb#L7) into the user model).

**Comment by lowellrex** [comment]
Exclude `JudgeTeam` admins from being able to visit the team admin page until @laurjpeterson can confirm that judges should be allowed to determine who is on their team.

**Comment by joeyyang** [comment]
discussed this approach with @sneha-pai — we thought it'd make more sense (and be much simpler to implement) if we put this link in the user's top menu dropdown, here:
![screenshot 2019-02-06 11 56 57](https://user-images.githubusercontent.com/2928395/52370153-0fd0f180-2a07-11e9-9dec-d42aa77f7a38.png)

Reasons:
- team management seems like a much less common action than switching queues, and we don't want to clutter the queue switcher with this action when it can be put somewhere less prominent
- it's much easier to implement this, especially when the top menu component already knows about who the user is, rather than passing the user object all the way down through the UI component tree

@laurjpeterson — can you approve these changes? I'm happy to work with you on a new set of AC.

**Comment by laurjpeterson** [comment]
@joeyyang this sounds good to me! Then, users can navigate back to their queue by clicking the Caseflow logo. I think we'll want to email about this change when it is deployed so they know the behavior.

In other but related news - @marvokdolor-gov got approval from DVCs today that judges _can_ manage the attorneys on their team, so they _can_ navigate to the user admin page. We were briefly told judges wouldn't be allowed to do this last week, and I alerted @tomas-nava and @lowellrex to it, but I'm glad to say that restricting judges from navigating is not, in fact, a requirement. 

I updated AC to reflect joey's comment and the recent news about judges.
